src/methods.js

```diff
@@ -98,8 +139,10 @@ const renderHTML = (testData, stylesheet) => new Promise((resolve, reject) => {
 		${testData.numPendingTests} pending
 	`);
 
+	const sortedTestResults = processTestSuiteResults(testData.testResults, config.getSort());
```

Comment:
Perhaps consider renaming this constant to `processedTestResults` as they are not always being exposed to the sorting method.